torchvision/datasets/cifar.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, root, train=True,
                  transform=None, target_transform=None,
                  download=False):
 
-        super().__init__(root, transform, target_transform)
+        super(VisionDataset, self).__init__(root, transform, target_transform)
```

Comment:
You should use the derived class name, so in this case `super(CIFAR10, self)`. Same applies to all other classes.

Reply:
Oops. Happens if you've never worked with Python2 ;)